angular.json

```diff
@@ -8,16 +8,23 @@
       "sourceRoot": "src",
       "projectType": "application",
       "prefix": "app",
-      "schematics": {},
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss"
+        },
+        "@schematics/angular:application": {
+          "strict": true
+        }
+      },
       "architect": {
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "outputPath": "dist/browser",
+            "outputPath": "dist/angular-checklist/browser",
```

Comment:
Any reason to change this path, it may impact the deployment.

Reply:
@santoshyadavdev Reverted back to original path 👍🏻

Reply:
No its not you, I think this path is used for deployment and may impact that.